asyncy/storyscript
# wrote a small function in version.py that deals with version flags in…

… command line (#211)



**- What I did**
Wrote a small function in version.py that addresses command line arguments looking for version flags (-v, --version, and version)
removed version flag handling in cli.py and replaced with function
**- How I did it**
Wrote function that compares command line arguments to the flags "-v", "--version" and "version" per issue #211 
call that function in cli.py to check user input arguments
**- How to verify it**
I did some manual testing to ensure it worked properly but most of the unit testes seemed broken and version test needs updating. Obviously, this would change how the version flag should be tested. I could write that test if need be.
**- Description for the changelog**
Modified version flags to accept -v, --version, and version. Closes #211
